sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -285,8 +285,8 @@ manifest(d = {
     "identifier": hex(CONST.ROM_EXT),
     "manuf_state_creator": hex(CONST.MANUF_STATE.PERSO_INITIAL),
     "visibility": ["//visibility:private"],
-    "version_major": ROM_EXT_VERSION.MAJOR,
-    "version_minor": ROM_EXT_VERSION.MINOR,
+    "version_major": "0xFFFFFFFF",
+    "version_minor": "0xFFFFFFFF",
```

Comment:
Should we leave these as-is and set the security version to max?  The security version is evaluated first and is not connected to any notion of "meaning" attached to the `major.minor` version.

This would allow inspection of the perso artifacts (e.g. `image manifest show`) to show a major/minor that has some relationship to when it was built.